manuscript/manuscript.Rmd

```diff
@@ -122,27 +122,33 @@ The goal of the agent is to learn how to maximize its expected cumulative reward
 The agent learns how to achieve this objective during a period called *training*.
 In training, the agent *explores* the available states and actions. 
 Once the agent comes across a highly rewarding sequence of states and actions, the agent will reinforce this behavior so that it is more likely for the agent to *exploit* the same high reward trajectory in future episodes.
-Throughout this process, the agent's behavior is codified into what is called a *policy*, which is a mapping of states to actions.
+Throughout this process, the agent's behavior is codified into what is called a *policy*, which describes what action an agent should take for a given state.
 
 To solve the RL problem, algorithms either take a *model-free* or *model-based* approach. 
-The distinction is that *model-free* algorithms do not have access to a model of the environment; yet, *model-based* algorithms use a model of the environment to achieve the RL objective.
+The distinction is that *model-free* algorithms do not attempt to learn or use a model of the environment.
+Conversely, *Model-based* algorithms either learn or use a model of the environment to achieve the RL objective.
 A trade-off between these approaches is that when it is possible to quickly learn a model of the environment or the model is already known, model-based algorithms tend to require much less interaction with the environment to learn good-performing policies [@mbpo].
 Yet, frequently, learning a model of the environment is very difficult, and in these cases, model-free algorithms tend to outperform [@CITE].
 Since most of the recent focus in RL has gone towards model-free algorithms, we will focus on model-free agents for the rest of the paper.
 
-Amongst model-free algorithms, there is divergence around what functions the agent is attempting to learn.
+Among model-free algorithms, there is divergence around what functions the agent is attempting to learn in order to achieve the RL objective.
```

Comment:
This section is good, but is getting more technical.  A discussion of value functions, policy gradients, and actor critic should all come *after* we introduce the basic notion of Bellman recursion and are discussing the details of the different algorithms.  

For the very high-level overview here I think we want to leave it at the level of the paragraph above, giving a heuristic feel for the idea of model-based vs model free.